src/Library/demos/List Model/main.blp

```diff
@@ -0,0 +1,101 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "List Model";
+  description: _("List models are a simple interface for ordered lists of GObject instances");
+
+  Box {
```

Comment:
Okay, so let's do this:

Except for the `StackSwitcher` and FlowBox items, go through and remove all the `halign: center` and `width-request` properties. Then put this `Box` inside an `Adw.Clamp` with `maximum-size: 640;` so you have something like:

```js
Adw.Clamp {
  maximum-size: 640;
  child:
    Box current_box {
      // ...
    };
}
```

Then set `hexpand: true` and `valign: start` on `list_box`, `flow_box` and `list_box_editable`. That should even up the whole demo.